.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,140 @@
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+name: Bug Report
+description: Report a bug
+
+body:
+  - type: checkboxes
+    attributes:
+      label: Contributing guidelines
+      description: |
+        Please read the contributing guidelines before proceeding.
+      options:
+        - label: I've read the [contributing guidelines](https://github.com/docker/buildx/blob/master/.github/CONTRIBUTING.md) and wholeheartedly agree
+          required: true
+
+  - type: checkboxes
+    attributes:
+      label: I've found a bug and checked that ...
+      description: |
+        Make sure that your request fulfills all of the following requirements.
+        If one requirement cannot be satisfied, explain in detail why.
+      options:
+        - label: ... the documentation does not mention anything about my problem
+        - label: ... there are no open or closed issues that are related to my problem
```

Comment:
This reads a little weird to me with the little paragraph in between:
![image](https://user-images.githubusercontent.com/7352848/185135540-f1136102-b590-45dd-b2ab-c985058ac4c8.png)

Not a blocker, but also not sure how to rephrase this to flow a little easier.